lib/modules/accountSwitcher.js

```diff
@@ -417,13 +414,13 @@ function _notifySwitchedAccountElsewhere() {
 	}
 
 	let message = username ?
-		`You switched to /u/${username}.` :
+		`${i18n("accountSwitcherUserSwitched",username)}` :
 		'You have been logged out.';
 
 	if (hasDraft) {
-		message += ` However, you haven't finished posting on this page as /u/${loggedInUser() || ''}.`;
+		message += ` ${i18n("accountSwitcherDraft", loggedInUser() || '')}`;
```

Comment:
FYI interpolation is fine here because prepending a space is awkward for translating (your code is correct)